### DISCUSSION TITLE
Access Value in VisualTree

### DISCUSSION THREAD

**Discussion by ollih1980** [AvaloniaUI/Avalonia#7208] [Q&A]
Hi all,

I would be great if somebody could tell me how to access a Value within the Visual Tree. Like in the DevTools.
e.g. I would like to read the Value from the Scrollbar of a DataGrid. Like this:

![image](https://user-images.githubusercontent.com/95570555/146671975-9e17b430-2e74-4493-8857-72d00b0ce7ed.png)

**Comment by ollih1980** [comment]
Any idea?

**Reply by kekekeks** [reply]
Traverse the visual tree using `IVisual.VisualParent` and `IVisual.VisualChildren`.
